examples/hazelcast/hazelcast-controller.yaml

```diff
@@ -0,0 +1,22 @@
+id: hazelcast
+kind: ReplicationController
+apiVersion: v1beta1
+desiredState:
+  replicas: 1
+  replicaSelector:
+    name: hazelcast
+  # This is identical to the pod config above
```

Comment:
delete this comment since there is no 'above' or modify to point to the other .yaml file.